Sources/App/Controllers/Heartbeat/HeartbeatController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  HeartbeatController.swift
+//  App
+//
+//  Created by di on 01.12.2017.
+//
+import Vapor
+import HTTP
+
+final class HeartbeatController  {
+  
+  func index(req: Request) throws -> ResponseRepresentable {
+    guard let beat = try Heartbeat.all().first else {
+      return Response.init(status: .noContent)
```

Comment:
No need to write `.init` here. `Response(status: .noContent)` - is good

Reply:
fixed